client/src/app/site/pages/meetings/pages/motions/modules/motion-poll/components/motion-poll-meta-information/motion-poll-meta-information.component.html

```diff
@@ -3,8 +3,7 @@
         {{ 'Groups' | translate }}:
 
         <span *ngFor="let group of poll.entitled_groups; let last = last">
-            {{ group.getTitle() | translate }}
-            <span *ngIf="!last">,</span>
+            {{ group.getTitle() | translate }}{{ !last ? ',' : '' }}
```

Comment:
I think
```suggestion
            <span>{{ group.getTitle() | translate }}</span><span>{{ !last ? ',' : '' }}</span>
```
is cleaner as it generally gives you more control.